aws-rds/aws-rds.cabal

```diff
@@ -139,8 +141,10 @@ test-suite test
     , text
   default-language: Haskell2010
   if flag(development)
-    ghc-options: -Werror -fplugin=SourceConstraints
-    build-depends:
-        source-constraints
+    ghc-options: -Werror
   else
     ghc-options: -Wwarn
+  if impl(ghc < 9.6)
```

Comment:
__question:__ Is there a ghc 9.5? If so, would we want to run the code on it? Or should/can this be `<= 9.4`?

Reply:
There is no 9.5. I will change it to `<= 9.4`. Good catch